.github/workflows/sycl-linux-precommit.yml

```diff
@@ -154,6 +154,21 @@ jobs:
             extra_lit_opts: --param matrix-xmx8=True --param gpu-intel-dg2=True
             env: '{"LIT_FILTER":${{ needs.determine_arc_tests.outputs.arc_tests }} }'
             use_igc_dev: true
+          - name: E2E tests on Intel Ponte Vecchio GPU
+            runner: '["Linux", "pvc"]'
+            env: '{"LIT_FILTER_OUT":"ESIMD/unified_memory_api/"}'
```

Comment:
if we remove this line do we still see timeouts even with the current number of jobs?

Reply:
And if so, `unsupported` via `local.lit.cfg.py` (or whatever it's named) is the way to do it.

Reply:
> if we remove this line do we still see timeouts even with the current number of jobs?

Let me try that out locally.

Reply:
> if we remove this line do we still see timeouts even with the current number of jobs?

I think we no longer need this line. With the current number of parallel jobs, I don't see spurious timeouts on PVC (in 15 test runs).

Reply:
ok cool lets remove it, these are important esimd tests so its good to enable them